src/pages/registration/resource_type_tab/sub_type_forms/research_data_types/RelatedResultItem.tsx

```diff
@@ -88,60 +80,54 @@ export const RelatedResultItem = ({
           </>
         )}
       </Box>
-      {isConfirmedDocument ? (
+      {isConfirmedDocument && registration ? (
```

Comment:
Denne vil vel føre til at loading under her aldri vil slå til?
`isLoadingRegistration` vil bare være `true` når vi ikke har `registration` enda, så linje 83 og 85 ser ut til å være gjensidig utelukkende

```suggestion
      {isConfirmedDocument ? (
```